kroxylicious-operator/examples/simple/03.VirtualKafkaCluster.my-cluster.yaml

```diff
@@ -15,4 +15,6 @@ spec:
     name: simple
   targetCluster:
     bootstrapping:
-      bootstrapAddress: my-cluster-kafka-bootstrap.kafka.svc.cluster.local:9092
\ No newline at end of file
+      bootstrapAddress: my-cluster-kafka-bootstrap.kafka.svc.cluster.local:9092
+  ingresses:
```

Comment:
Should this be `ingressRefs`, to follow the same pattern we've established for references to other resources?